README.md

```diff
@@ -90,43 +111,73 @@ worker.get('virtual/.', function(request, response) {
 });
 ```
 
-The output for any request that hits this url will be the original address of the url and a random number, with the headers that we specified in the response object.
+The output for any request that hits this URL will be the original address of the URL, and a randomly generated number. Note how we can even specify the headers for our response!
+
+Remember that the handler *must always return a Promise*.
+
+### onMessage(?) TODO
```

Comment:
Will receive this event if an actor (window, iframe, another worker or sharedworker) perform a `postMessage` to the ServiceWorker to communicate with it.